lib/browser/objects-registry.js

```diff
@@ -20,6 +20,7 @@ class ObjectsRegistry {
   add(webContents, obj) {
     // Get or assign an ID to the object.
     let id = this.saveToStorage(obj);
+    this.storage[id].count++;
```

Comment:
Ah, because each `variable` isn't individually garbage collected, but rather the value they point to, always.
I learned something about V8 today! :+1:
Thanks for explaining!